config/goerli.json

```diff
@@ -0,0 +1,68 @@
+{
+  "rpcURL": "https://kovan.infura.io/v3/11465e3f27b247eb8b785c23047b29fd",
```

Comment:
this is for Goerli is it not ? You will need and rpc endpoint to goerli netowork